menas/ui/service/GenericService.js

```diff
@@ -104,4 +104,11 @@ var GenericService = new function() {
 
         fnHelper("", sModelAcc, paths)
     };
+
+    this.validEntityName = function(sName) {
```

Comment:
`validateEntityName` not `validEntityName`

Reply:
My bad :-(

Reply:
resolved